lib/fetch.js

```diff
@@ -55,6 +54,32 @@ export function mapAndSanitate(item, type) {
   };
 }
 
+export function sanitize(key, type, defaultValue, json) {
```

Comment:
`ramda/curry`. Don't think there's a need to curry unless actually needed, however.